src/applications/beta-enrollment/beta-enrollment-entry.jsx

```diff
@@ -4,9 +4,21 @@ import startApp from '../../platform/startup';
 
 import routes from './routes';
 import manifest from './manifest.json';
+import { REGISTER_SERVICE } from './actions';
+
+const analyticsEvents = [
+  {
+    action: REGISTER_SERVICE,
+    event: (store, action) => ({
+      event: 'beta-enrollment',
+      appName: action.service,
```

Comment:
The current implementation of the analytics middleware expects this function to return a string. The middleware will need to be tweaked a bit to accept an object. Otherwise, this looks good.